tests/misc/test_basic_without_ssh.py

```diff
@@ -72,6 +73,18 @@ def test_checkpoint(self, imported_vm):
         finally:
             snapshot.destroy(verify=True)
 
+    def test_fileserver_redirect_https(self, host):
```

Comment:
This file is not the best place to add this test: it's a collection of tests that run with *a VM* (without using SSH).

We'll need a new test file in misc.